src/test_l1_tvSettings.c

```diff
@@ -337,19 +336,19 @@ void test_l1_tvSettings_positive_RegisterVideoFormatChangeCB (void)
 
 	/* Step 02: Calling tvsettings RegisterVideoFormatChangeCB and expecting the API to return success*/
 	callbackData.userdata = (char *)malloc(10);
-        if(callbackData.userdata){
-                strcpy(callbackData.userdata, "TestData");
-        }
+	if(callbackData.userdata){
+		strcpy(callbackData.userdata, "TestData");
+	}
 	callbackData.cb = tvVideoFormatChangeHandler;
-	
+
 	result = RegisterVideoFormatChangeCB(&callbackData);
 	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_NONE);
```

Comment:
can you rename this please, `UT_ASSERT_AUTO_TERM_NUMERICAL` - is not part of UT, therefore not a valid name.

Reply:
This cleanup is planned as part of https://github.com/rdkcentral/rdkv-halif-test-tvsettings/issues/18

Reply:
I've added that to #18 then because it wasn't mentioned in requirements.